easybuild/easyconfigs/l/libcifpp/libcifpp-7.0.8-GCC-13.3.0.eb

```diff
@@ -0,0 +1,59 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2025/01
+easyblock = 'CMakeMake'
+
+name = 'libcifpp'
+version = '7.0.8'
+
+homepage = 'https://github.com/PDB-REDO/libcifpp'
+description = """This library contains code to work with mmCIF and PDB files"""
+
+toolchain = {'name': 'GCC', 'version': '13.3.0'}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['2297e6649a4f71caf9da5f1d97f59512e7324bb62083bb5b08eb00c1c0385cb3']
+
+github_account = 'PDB-REDO'
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+    ('Eigen', '3.4.0'),
+    ('Boost', '1.85.0'),  # req. for tests
+    ('mrc', '1.3.13'),
+    ('Catch2', '2.13.10')
+]
+dependencies = [
+    ('zlib', '1.3.1'),
+]
+
+# CIFPP_DOWNLOAD_CCD is set to OFF in order to allow for consistent module rebuild.
+# Therefore, ciffpp data shipped with the src is installed into %(builddir)s/shared/libcifpp without an 
+# online update at build time.
+# However, "The files in CIFPP_DATA_DIR are quickly becoming out of date". The data can be installed elsewhere
+# outside the module directory in order to regularly run the update script - in this case
+# the variable CIFFPP_DATA_DIR and LIBCIFPP_DATA_DIR require to be adapted accordingly.
+# More information about the update script can be found here:
+# https://github.com/PDB-REDO/libcifpp#building
+
+_confopts = '-DFETCHCONTENT_FULLY_DISCONNECTED=ON '
+# do not update components.cif:
+_confopts += '-DCIFFPP_DOWNLOAD_CCD=OFF '
+
+configopts = [
+    '-DBUILD_SHARED_LIBS=OFF ' + _confopts,
+    '-DBUILD_SHARED_LIBS=ON ' + _confopts,
+]
+
+runtest = True
+
+sanity_check_paths = {
+    'files': ['lib/libcifpp.%s' % x for x in ['a', SHLIB_EXT]],
+    'dirs': ['etc',	'include', 'lib', 'lib/cmake', 'share/%(name)s']
+}
+
+modextrapaths = {
+    'CIFPP_DATA_DIR': 'share/libcifpp',
+    'LIBCIFPP_DATA_DIR': 'share/libcifpp/'
```

Comment:
I assume that there is a reason for the trailing / here? If not drop it.

Reply:
not sure. However, I removed it